desc/compute/_bootstrap.py

```diff
@@ -312,7 +312,7 @@ def j_dot_B_Redl(
 @register_compute_fun(
     name="<J*B> Redl",
     label="\\langle\\mathbf{J}\\cdot\\mathbf{B}\\rangle_{Redl}",
-    units="T A m^{-2}",
+    units="N / m^{3}",
```

Comment:
I prefer the units as T*A/m^2. Although this has units of Newtons, its not really a force